tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -108,6 +108,9 @@ ROOT::Experimental::RNTupleReader::RNTupleReader(
    if (!fModel) {
       throw RException(R__FAIL("null model"));
    }
+   if (!fModel->GetProjectedFields().IsEmpty()) {
+      throw RException(R__FAIL("model has projected fields, which is incompatible with providing a read model"));
```

Comment:
This sounds weird (I am probably missing the bigger context) as this sounds like projected field can only be used when writing (where as they sound like they would mainly be useful when reading).